xonsh/parsers/completion_context.py

```diff
@@ -27,32 +27,47 @@
 
 
 class CommandArg(NamedTuple):
+    """An argument for a command"""
+
     value: str
```

Comment:
can this be replaced with Attributes section?

Reply:
What do you mean?

Reply:
https://numpydoc.readthedocs.io/en/latest/format.html#class-docstring
Like here mentioned Attributes section

Reply:
Oh, no because this is how `NamedTuples` are used

Reply:
Good then